packages/flame_forge2d/lib/sprite_body_component.dart

```diff
@@ -9,9 +9,11 @@ abstract class SpriteBodyComponent<T extends Forge2DGame>
   /// body that is create in createBody()
   SpriteBodyComponent(
     Sprite sprite,
-    Vector2 spriteSize,
-  ) : super(
-          positionComponent: SpriteComponent(size: spriteSize, sprite: sprite),
+    Vector2 spriteSize, {
+    int priority = 0,
```

Comment:
This should be nullable and not have a default (the default is set in `Component`).